config.yaml

```diff
@@ -81,6 +81,92 @@ menu:
       pre: '<i class="fa fa-heart fa-sm"></i>&nbsp;'
       params:
         anchorClass: 'btn btn-danger btn-sm mx-4'
+  footer_one:
+    - identifier: features
+      name: Features
+      weight: 10
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Features
+    - identifier: demo
+      name: Demo
+      weight: 20
+      url: demo
+    - identifier: download 
+      name: Download
+      weight: 30
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Downloads
+    - identifier: modules
+      name: Modules
+      weight: 40
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Modules
+    - identifier: blog
+      name: Blog
+      weight: 50
+      url: blog
+    - identifier: docs
+      name: Docs
+      weight: 60
+      url: https://open-emr.org/wiki
+    - identifier: forum
+      name: Forum
+      weight: 70
+      url: https://community.open-emr.org/
+    - identifier: chat
+      name: Chat
+      weight: 80
+      url: chat
+    - identifier: donate
+      name: Donate
+      weight: 90
+      url: donate
+    - identifier: support
+      name: Support
+      weight: 100
+      url: support
+  footer_two:
+    - identifier: faq
+      name: FAQ
+      weight: 10
+      url: https://open-emr.org/wiki/index.php/FAQ
+    - identifier: forum
+      name: Forum
+      weight: 20
+      url: https://community.open-emr.org/
+    - identifier: user_manuals
+      name: User Manuals
+      weight: 30
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Wiki_Home_Page#User_Manual
+    - identifier: support_guide
+      name: Support Guide
+      weight: 40
+      url: /support
+    - identifier: professional_support
+      name: Professional Support
+      weight: 50
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Professional_Support
+    - identifier: onc
+      name: Certification
+      weight: 60
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Wiki_Home_Page#ONC_Ambulatory_EHR_Certification
```

Comment:
this is not going to the ONC section

Reply:
https://github.com/openemr/website-openemr/commit/6ac80a9323aabc4f8b6f4e4e567bce73444d1baf